fwbot/tests/common.py

```diff
@@ -0,0 +1,86 @@
+import random
+from typing import Dict
+
+from odoo.tests.common import TransactionCase
+
+from ..models.merge_request import MergeRequest
+from ..models.repository import Repository
+
+
+class MockResponse:
```

Comment:
Why not using request-mock?

Reply:
Wanted to keep outside dependencies as less as possible.